### DISCUSSION TITLE
[Idea] hardware whitelist

### DISCUSSION THREAD

**Discussion by martiou** [glpi-project/glpi-agent#485] [Ideas]
### Your idea

We have teleworking devices belonging to our employees. They should not appear in our company inventory. 
Would it be possible to draw up a white list accepted for the synchronization of screens, webcams and other known peripherals (brand, model, etc.) in our company?
The synchronization agent would then no longer report devices not accepted in our inventory.

**Comment by g-bougard** [comment]
Hi @martiou 
this question is indeed a GLPI question.
The answer is you can create rules in the "Rules for assigning an item and link equipments" in GLPI Administration to filter out unwanted devices. Have you tried it ?
